lib/src/widgets/camera_picker.dart

```diff
@@ -742,12 +742,16 @@ class CameraPickerState extends State<CameraPicker>
 
     if (controller.value.isRecordingVideo) {
       controller.stopVideoRecording().then((XFile file) async {
-        final AssetEntity? entity = await _pushToViewer(
-          file: file,
-          viewType: CameraPickerViewType.video,
-        );
-        if (entity != null) {
-          Navigator.of(context).pop(entity);
+        if (widget.pickerConfig.onStopRecordingVideo != null) {
```

Comment:
We've already defined a getter at:
https://github.com/fluttercandies/flutter_wechat_camera_picker/blob/16dc2cdd9afc5a5685071e7a84f93bb6d9e7456c/lib/src/widgets/camera_picker.dart#L227

`widget.config` is not necessary here.

Reply:
Thanks and resolved